lib/models/pr.js

```diff
@@ -153,70 +155,74 @@ class PR {
     get payload() {
         return this._payload;
     }
-        
+
     set payload(payload) {
         return this._payload = payload;
     }
 
     get installation() {
         return this._installation;
     }
-    
+
     get number() {
         return this._number;
     }
-    
+
     get owner() {
         return this._owner;
     }
-    
+
     get repo() {
         return this._repo;
     }
-    
+
     get id() {
         return this._id;
     }
-    
+
     get body() {
         return this._body = this._body || (this.payload.body || "").replace(/\r\n/g, "\n").trim();
     }
-    
+
     get processor() {
         return this.config.type.toLowerCase();
     }
-    
+
     get aws_s3_bucket() {
         if (process.env.ALLOW_MULTIPLE_AWS_BUCKETS == "no") {
             return process.env.AWS_BUCKET_NAME;
         }
         return this.owner == "whatwg" ? process.env.WHATWG_AWS_BUCKET_NAME : process.env.AWS_BUCKET_NAME;
     }
-    
+
     get isMultipage() {
         return !!this.config.multipage;
     }
 
     requiresPreview() {
         return !(/<!--\s*no preview\s*-->/.test(this.body));
     }
-    
+
+    async requestIncludes() {
+        this.includes = await scanIncludes(Head.fromPR(this).github_repo_url, this.config.src_file, this.config.type);
+    }
+
     touchesSrcFile() {
         let relFiles = this.relevantSrcFiles;
         return this.files.some(f => relFiles.indexOf(f.filename) > -1);
     }
-    
+
     get relevantSrcFiles() {
-        return [this.config.src_file];
+        return [this.config.src_file].concat(this.includes);
```

Comment:
nit: 
```suggestion
        return [].concat(this.config.src_file, this.includes);
```

Reply:
I'd somewhat expected that to iterate the letters in `src_file`, but it works. However, it turns out that `this.includes` both includes `src_file` on its own, and is a `Set`, so this line didn't work at all. Fixed.

I'm not sure what style you want here, @tobie. I could just assign to `this.relevantSrcFiles` and remove the getter?